### PR TITLE
III-5079  -  Links to tabs in OfferScore

### PR DESCRIPTION
**Pull request by brampauwelyn**
### Added
- Links to tabs in OfferScore
Screenshot 2023-02-21 at 09 35 46

---
Ticket: https://jira.uitdatabank.be/browse/III-5079

### CODE REVIEW

File: src/pages/steps/AdditionalInformationStep/OfferScore.tsx [hunk 1]
```diff
@@ -184,7 +191,7 @@ const OfferScore = ({ completedFields, offerId, scope, ...props }: Props) => {
     return maxRotation * scorePercentage + minRotation;
   }, [score]);
 
-  const tip = useMemo(() => {
+  const tipField = useMemo(() => {
     if (score === 100)
       return t(`create.additionalInformation.event_score.tip.completed`);
```
Comment (Anahkiasen) [inline src/pages/steps/AdditionalInformationStep/OfferScore.tsx:196]: This branch of the condition isn't used anymore I think

File: src/pages/steps/AdditionalInformationStep/OfferScore.tsx [hunk 2]
```diff
@@ -211,9 +218,27 @@ const OfferScore = ({ completedFields, offerId, scope, ...props }: Props) => {
...
+    return (
+      <Link
+        color={getValue('link')}
+        href={`#${hash}`}
+        onClick={(e) => {
+          e.preventDefault();
+          router.push({ hash }, undefined, {
+            shallow: true,
+          });
+        }}
```
Comment (Anahkiasen) [inline src/pages/steps/AdditionalInformationStep/OfferScore.tsx:235]: Is the `href` needed if it's overriden and prevented, or is it purely for ESLint purposes?

File: src/pages/steps/AdditionalInformationStep/OfferScore.tsx [hunk 3]
```diff
@@ -233,7 +258,19 @@ const OfferScore = ({ completedFields, offerId, scope, ...props }: Props) => {
...
+        <Text>
+          {score === 100 &&
+            t('create.additionalInformation.event_score.tip.completed')}
+          {score !== 100 && (
+            <Trans
+              i18nKey={`create.additionalInformation.event_score.tip.${tipField}.text`}
+            >
+              Tip: <TipLink field={tipField} />
+            </Trans>
+          )}
```
Comment (Anahkiasen) [inline src/pages/steps/AdditionalInformationStep/OfferScore.tsx:271]: Could maybe a do a ternary here

File: src/pages/steps/EventTypeAndThemeStep.tsx [hunk 4]
```diff
@@ -205,6 +206,9 @@ const EventTypeAndThemeStep = ({
   shouldHideType,
 }: Props) => {
   const { t, i18n } = useTranslation();
+  const { asPath } = useRouter();
+  const [_path, hash] = asPath.split('#');
```
Comment (Anahkiasen) [inline src/pages/steps/EventTypeAndThemeStep.tsx:210]: In theory `[, hash]` should be valid syntax

**Review by Anahkiasen — APPROVED** [review]
Some small tweak suggestions but nothing blocking so looks all good to me

File: src/pages/steps/AdditionalInformationStep/OfferScore.tsx (hunk 1 above)
Comment (brampauwelyn) [reply src/pages/steps/AdditionalInformationStep/OfferScore.tsx:196]: is it possible this was an earlier commit?
It should already be `tipField`

File: src/pages/steps/AdditionalInformationStep/OfferScore.tsx (hunk 2 above)
Comment (brampauwelyn) [reply src/pages/steps/AdditionalInformationStep/OfferScore.tsx:235]: Yes purely for esLint purpose

**Pull request merged**